lib/surrealist/schema_definer.rb

```diff
@@ -15,7 +15,11 @@ class SchemaDefiner
     def self.call(klass, hash)
       raise Surrealist::InvalidSchemaError, 'Schema should be defined as a hash' unless hash.is_a?(Hash)
 
-      klass.instance_variable_set('@__surrealist_schema', hash)
+      if klass.name =~ /ROM::Struct/
+        klass.class_variable_set('@@__surrealist_schema', hash)
```

Comment:
This will probably cause undesired behavior with ROM(?)

Probably need similar spec for ROM similar to this one.
https://github.com/nesaulov/surrealist/blob/0a8a7ad36346bbe2d3030e6968e6e92b2dc6ca6e/spec/build_schema_spec.rb#L194-L19811

I guess this brings up the problem of what is the minimum expected behaviour we want to test since we are not fully replicating all tests for each ORM.

Reply:
I've added such spec. Talking about minimal behaviour to test, I think that ROM is pretty much covered right now, though it may seem that there are not so many examples there. AR tests are pretty massive because of scopes